### DISCUSSION TITLE
Noob question: available for .ps1 files ?

### DISCUSSION THREAD

**Discussion by bro69noscope** [HiPhish/rainbow-delimiters.nvim#134] [Q&A] [answered]
Just wondering if the plugin is supposed to work on powershell lang too. Treesitter syntax highlithing works just fine, but I don't see any pretty delimiters: been trying a few things here and there but I'm wondering if I may just be wasting my time. 😅

**Comment by HiPhish** [comment, marked as answer]
Power shell is not supported. It could be if someone were to write the queries, but no one has done so yet. You can find out which languages are supported by looking into the queries directory. PRs are always welcome. See :h rb-delimiters-custom-queries for instructions on writing new queries.
